### PR TITLE
SAK-43280 Lessons: Answer text input is unusable if the section is co…

### PR DESCRIPTION
**Pull request by mpellicer**
…llapsed.

### CODE REVIEW

File: lessonbuilder/tool/src/webapp/templates/ShowPage.html [hunk 1]
```diff
@@ -888,7 +888,7 @@ <h3 class="questionText" rsf:id="questionText"></h3>
 				      <input type="hidden" rsf:id="csrf5" />
                                         <input type="hidden" rsf:id="shortanswerId" />
                                         <div rsf:id="msg=simplepage.answer"></div>
-                                        <textarea rsf:id="shortanswerInput" class="resizable" ></textarea>
+                                        <textarea rsf:id="shortanswerInput"></textarea>
```
[the remark this answers is not in the document]
Comment (adrianfish) [reply lessonbuilder/tool/src/webapp/templates/ShowPage.html]: I'd stick an aria-labelledby on the textarea here. I think the div with simplepage.answer is the label?

[the remark this answers is not in the document]
Comment (mpellicer) [reply lessonbuilder/tool/src/webapp/templates/ShowPage.html]: Ok

**Comment by jesusmmp** [comment]
LGTM!

File: lessonbuilder/tool/src/webapp/templates/ShowPage.html [hunk 2]
```diff
@@ -887,8 +887,8 @@ <h3 class="questionText" rsf:id="questionText"></h3>
                                     <form action="#" rsf:id="shortanswerForm" class="shortanswerForm">
 				      <input type="hidden" rsf:id="csrf5" />
                                         <input type="hidden" rsf:id="shortanswerId" />
-                                        <div rsf:id="msg=simplepage.answer"></div>
-                                        <textarea rsf:id="shortanswerInput" class="resizable" ></textarea>
+                                        <div for="answerInput" rsf:id="msg=simplepage.answer"></div>
+                                        <textarea rsf:id="shortanswerInput" aria-labelledby="answerInput"></textarea>
```
Comment (adrianfish) [inline lessonbuilder/tool/src/webapp/templates/ShowPage.html]: I don't think this is going to work. for="answerInput" should be id="answerInput" I think.

Comment (mpellicer) [reply lessonbuilder/tool/src/webapp/templates/ShowPage.html]: Ok, this should work.

**Pull request merged**